README.md

```diff
@@ -1,2 +1,90 @@
 IIIF image API implemented using Fastly IO
 ==========================================
+
+[![Build Status](https://travis-ci.org/elifesciences/fastly-iiif.svg?branch=master)](https://travis-ci.org/elifesciences/fastly-iiif)
+
+This library provides an implementation of the [IIIF image API](https://iiif.io/) using [Fastly Image Optimizer](https://www.fastly.com/products/web-and-mobile-performance/image-optimization)
+
+Supported versions
+------------------
+
+- [2.1](https://iiif.io/api/image/2.1/) (level 0), with:
+  - Formats:
+    - `jpg`
+  - Features:
+    - `cors`
+
+Installation
+------------
+
+> You will need a Fastly service with VCL and image optimization enabled.
+
+1. Create a new configuration version.
+2. Set up a domain and host.
+3. Upload [`iiif.vcl`](iiif.vcl) as the 'main' [custom VCL](https://docs.fastly.com/vcl/custom-vcl/uploading-custom-vcl/).
+4. Upload a [configuration snippet](#configuration).
+5. Add any custom settings and snippets as required.
+6. Activate the version.
```

Comment:
`7. Wait for request ... to return the header ... to confirm version has been rolled out`

Reply:
That header's part of the test suite only.